static/install.html

```diff
@@ -82,6 +83,31 @@ <h2 id="prerequisites">
             from the <a href="https://developers.google.com/android/ota">full update package
             page</a>.</p>
 
+            <h3 id="adding-udev-rules">
+                <a href="#adding-udev-rules">Adding udev rules</a>
+            </h3>
+            <p>Linux distributions that use udev for device management, such as Fedora, Ubuntu,
+            and Debian do not include and require correct udev rules in order to permit the
+            software on the computer to talk to the device if running adb or fastboot as a non-
+            root user. Running fastboot as non-root without the prerequisite udev rules installed
+            will result in fastboot hanging with a <code>waiting for device</code> error message
+            even with the device plugged in and seemingly properly configured.</p>
+
+            <p>The easiest way to obtain and add the correct udev rules to your computer is to
+            run the following commands to download the android rules, move them to the udev rules
+            directory and reload the rules, then add the <code>adbusers</code> usergroup and
```

Comment:
Need to remove this bit too.

Reply:
Whoops. I removed the part about the adbusers usergroup but left the intro paragraph and the Fedora and Debian.